repos/system_upgrade/el7toel8/actors/excluderepositories/libraries/excluderepositories.py

```diff
@@ -0,0 +1,145 @@
+from leapp import reporting
+from leapp.libraries.common.config import get_product_type
+from leapp.libraries.stdlib import api
+from leapp.models import (
+    CustomTargetRepository,
+    RepositoriesExcluded,
+    RepositoriesFacts,
+    RepositoriesMap,
+)
+
+manually_enabled_repos_in_use = []
```

Comment:
This global var is quite confusing. What is it supposed to hold? All the repoids passed through the `--enablerepo` option?
It is hard for me to untangle it's purpose even if I find the place where this var is being filled. From what I understand, it should be either empty or holds **just** the CRB repo. But it's hard for me to read the code to verify that.
Instead of a global var, I'd rather see a function with a proper name that returns this list.

For example, it could be like this in `process()`:
```python
repos_to_exclude = _get_repos_to_exclude()
manually_enabled_repos = _get_manually_enabled_repos()
# The manually enabled repos override the default exclusion
repos_to_exclude = list(set(repos_to_exclude) - set(manually_enabled_repos))
overriden_repos_to_exclude = list(set(repos_to_exclude) & set(manually_enabled_repos)) 
_report_using_unsupported_repos(overriden_repos_to_exclude)
```